node/src/BaseClient.ts

```diff
@@ -1241,6 +1242,32 @@ export class BaseClient {
         return this.createWritePromise(createSCard(key));
     }
 
+    /** Gets the intersection of all the given sets.
+     * See https://valkey.io/docs/latest/commands/sinter/ for more details.
+     *
+     * @remarks When in cluster mode, all `keys` must map to the same hash slot.
+     * @param keys - The `keys` of the sets to get the intersection.
+     * @returns A set of members which are present in all given sets.
```

Comment:
Do we need a dash here?
```suggestion
     * @returns - A set of members which are present in all given sets.
```

Reply:
Very good point. I see some methods' returns have dashes and some methods' returns do not have dashes. Should I just add a dash?

Reply:
Oh, in that case I'd add this dash and publish another PR which updates other docs to align the style.